makefiles/color.inc.mk

```diff
@@ -1,10 +1,26 @@
+# Set colored output control sequences if the terminal supports it and if
+# not disabled by the user
+
 COLOR_GREEN  :=
 COLOR_RED    :=
 COLOR_PURPLE :=
 COLOR_RESET  :=
 COLOR_ECHO   := /bin/echo
 
-ifeq (0,  $(shell tput colors 2>&1 > /dev/null; echo $$?))
+ifeq ($(CC_NOCOLOR),)
+  available_colors:=$(shell tput colors)
```

Comment:
`tput` can still fail when the terminfo file is borked. This would lead to some error messages during the invokation of `Make`. Could you redirect stderr to `/dev/null` like before? Can be verified with: `TERM="borked" tcap colors`.

Reply:
Personally when errors occur I prefer to see the reason, rather than have the error message suppressed.  However I've put in the redirect as you requested, and pushed  a new commit.

Reply:
@fetmos I can see your concerns about error visibility. But when `tput` fails because of completely different reasons and your shell is otherwise in a valid state, then it will get pretty confusing especially for non advanced RIOT users. Since this check is just deciding about a colored output, I guess we can go with this way. Thanks for the update!